core/src/main/java/com/dtp/core/support/selector/RandomExecutorSelector.java

```diff
@@ -0,0 +1,22 @@
+package com.dtp.core.support.selector;
+
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.Executor;
+
+/**
+ * RandomExecutorSelector related
+ *
+ * @author yanhom
+ * @since 1.1.3
+ **/
+public class RandomExecutorSelector implements ExecutorSelector {
+
+    private final Random random = new Random(System.currentTimeMillis());
```

Comment:
I think it would be better to use ThreadLocalRandom

Reply:
I think so